releases/1.0.0/index.md

```diff
@@ -33,12 +33,14 @@ custom_css: releases
 * Added OWL 2 DL review with SHACL-SPARQL for UCO ([*Github Pull Request 102*](https://github.com/casework/CASE/pull/102))
   * (While this is not necessarily a backwards-incompatible change within the scope of UCO, it does identify what end users might not have realized were broader OWL errors.)
 * Removed CASE's IRI base ([*GitHub Pull Request 103*](https://github.com/casework/CASE/pull/103)) 
+* Removed requirement of repeating all property constraints from parent to child classes ([*UCO GitHub Issue 417*](https://github.com/ucoProject/UCO/issues/417))
 
 #### General changes
 
 *(These are general changes to the preexisting ontology that are not breaking or range changes.)*
 
 * Fixed Makefile to grab all of UCO's files, even those under different directories ([*GitHub Pull Request 101*](https://github.com/casework/CASE/pull/101))
+* Changed minCount from 1 to 0 on multiple properties ([*UCO GitHub Issue 428*](https://github.com/ucoProject/UCO/issues/428))
```

Comment:
CASE and UCO each have their own Issue for this.  CASE's is [Issue 98](https://github.com/casework/CASE/issues/98).